<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1060</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1060</width>
    <height>540</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Logosímbolo-vertical.jpg</normaloff>Logosímbolo-vertical.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
border-radius:10px;
background-image: url(icon/c.png)
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>511</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox#comboBox{
background-color: rgb(177, 100, 182);
font: 14pt &quot;Comic Sans MS&quot;;
border-radius: 15px;
}
QComboBox#comboBox:hover{
background-color: rgba(177, 100, 182,120);
font: 14pt &quot;Comic Sans MS&quot;;
border-radius: 15px;
}
QComboBox#comboBox:pressed{
background-color: rgba(177, 100, 182,140);
font: 14pt &quot;Comic Sans MS&quot;;
border-radius: 15px;
}</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="lista">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>420</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>421</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>9</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="midLineWidth">
     <number>10</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>5</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>25</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Atributos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salir">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#salir{
background-color: rgb(177, 100, 182);
font: 14pt &quot;Comic Sans MS&quot;;
border-radius: 15px;
}
QPushButton#salir:hover{
background-color: rgba(177, 100, 182,120);
font: 14pt &quot;Comic Sans MS&quot;;
border-radius: 15px;
}
QPushButton#salir:pressed{
background-color: rgba(177, 100, 182,140);
font: 14pt &quot;Comic Sans MS&quot;;
border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>salir</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>501</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>501</width>
      <height>22</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>491</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>491</width>
      <height>351</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>491</width>
      <height>351</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Logosímbolo-vertical.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>260</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/fon/icon/HK2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="tipo_corte">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox#tipo_corte{
background-color: rgb(177, 100, 182);
font: 14pt &quot;Comic Sans MS&quot;;
border-radius: 15px;
}
QComboBox#tipo_corte:hover{
background-color: rgba(177, 100, 182,120);
font: 14pt &quot;Comic Sans MS&quot;;
border-radius: 15px;
}
QComboBox#tipo_corte:pressed{
background-color: rgba(177, 100, 182,140);
font: 14pt &quot;Comic Sans MS&quot;;
border-radius: 15px;
}</string>
    </property>
    <property name="currentText">
     <string>Axial</string>
    </property>
    <item>
     <property name="text">
      <string>Axial</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sagital</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Coronal</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="HK.qrc"/>
  <include location="fond.qrc"/>
 </resources>
 <connections/>
</ui>
